File: app/Models/Recording.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Recording extends Model
{
    protected $fillable = [
        'course_id',
        'file_url',
        'duration',
    ];

    public function course()
    {
        return $this->belongsTo(Course::class);
    }
}

File: applicationKT/app/src/main/java/com/englishapp/data/model/Recording.kt
package com.englishapp.data.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Recording(
    @Json(name = "id") val id: Long,
    @Json(name = "course_id") val courseId: Long,
    @Json(name = "file_url") val fileUrl: String,
    @Json(name = "duration") val duration: Int? = null,
    @Json(name = "course") val course: Course? = null,
    @Json(name = "created_at") val createdAt: String? = null,
    @Json(name = "updated_at") val updatedAt: String? = null
)
